test/test_surface_fluxes.jl

```diff
@@ -80,16 +81,13 @@ end
     # the atmosphere, have zero gustiness and a constant roughness length of
     # `1e-4` for momentum, water vapor and temperature
     # For this case we can compute the fluxes by hand.
-
-    @inline neutral_bulk_coefficients(ψ, h, ℓ, L) = log(h / ℓ)
-
     ℓ = 1e-4
 
     roughness_lengths = SimilarityScales(ℓ, ℓ, ℓ)
     similarity_theory = SimilarityTheoryTurbulentFluxes(grid; 
                                                         roughness_lengths, 
                                                         gustiness_parameter = 0,
-                                                        bulk_coefficients = neutral_bulk_coefficients)
+                                                        similarity_profile_type = NeutralLogarithmicSimilarityProfile())
```

Comment:
Why not use a stability function that returns 0. Isn't this simpler?

Reply:
Probably though, we should not have written the code to allow this kind of degeneracy and the "stability function" and "similarity profile" should be combined.